src/style.css

```diff
@@ -1,3 +1,63 @@
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Consider specifying a fallback font-family in case 'Roboto' isn't available. This is a good practice to ensure text is always displayed with a suitable font. For example: `font-family: Roboto, 'Arial', sans-serif;`